sys.py/UI/skin_manager.py

```diff
@@ -23,52 +23,55 @@ class SkinManager(object):
     _Config = None
     
     def __init__(self):
-        pass
+        self.Init()
 
     def ConvertToRGB(self,hexstr):
         
         h = hexstr.lstrip('#')
         return tuple(int(h[i:i+2], 16) for i in (0, 2 ,4))
     
     def Init(self):
-        
+        if not SkinManager._Colors:
```

Comment:
This keeps only one skin manager colors active.
When we add a skin switcher we can call "SetColors" to change the global color scheme
(although, most likely, it will be easier to just restart the launcher after updating the config)